css/style.css

```
@@ -0,0 +1,781 @@
+/* 共通 */
+
+html {
+  font-size: 62.5%;
+  letter-spacing: 0.05em;
+}
+
+li {
+  list-style-type: none;
+}
+
+a {
+  text-decoration: none;
+}
+
+body {
+  font-family: "Noto Serif", "Noto Serif JP", serif;
+  word-break: break-all;
+}
+
+*,
+*:before,
+*:after {
+  -webkit-box-sizing: border-box;
+  box-sizing: border-box;
+}
+
+/* ヘッダー */
+.inner {
+  max-width: 1200px;
+  margin: 0 auto;
+}
+
+/* ヘッダー色変更 */
+.header-inner {
+  background-color: #282f35;
+  transition: all 0.5s;
+  width: 100%;
+  position: fixed;
+  left: 0;
+  right: 0;
+  z-index: 100;
+}
+
+.hidden {
+  background: none;
+  transition: all 0.5s ease;
+}
+
+.header-list {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin: 0 auto;
+  padding: 20px 0;
+}
+
+.header-title h1 {
+  font-size: 2.6rem;
+  line-height: 1.375;
+  font-weight: bold;
+  font-style: italic;
+  color: #ffffff;
+}
+
+.nav-list {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.nav-list-item {
+  font-size: 1.6rem;
+  line-height: 1.375;
+  font-weight: bold;
+  font-style: italic;
+  margin-right: 30px;
+  text-align: center;
+}
+
+.nav-list-item:hover {
+  opacity: 0.8;
+}
+
+.nav-list-item a {
+  color: #ffffff;
+}
+
+.nav-list-item:last-child {
+  margin-right: 0;
+}
+
+.fv {
+  background-image: url(../img/fv-bgi_01@2x.jpg);
+  height: 750px;
+  background-repeat: no-repeat;
+  background-position: center;
+  background-size: cover;
+}
+

```

画像が切り替わるときにバチバチ切り替わるのが気になります。
そこで提案ですが,簡単に` transition`させることで自然な動きになると思いますがいかがでしょうか？（指定数値は適当です。）
```suggestion
.fv {
  background-image: url(../img/fv-bgi_01@2x.jpg);
  height: 750px;
  background-repeat: no-repeat;
  background-position: center;
  background-size: cover;
  transition: all  1s ease;
}
```

---

ありがとうございます。
気になってたのですが、jsでの実装がわからず変更できずにいました。
教えて頂いた内容で動きもとても自然だと感じました。
下記コミットで修正しました。
e447772